terra_notebook_utils/table.py

```diff
@@ -62,14 +62,19 @@ def _init_request_data(self):
 
     def _get_row_update_request_data(self, row: Row) -> List[Dict[str, Any]]:
         request_data = list()
+        types: Set[type]
         for name, val in row.attributes.items():
             if isinstance(val, str):
                 update_ops = list()  # No Firecloud update operations needed for string values
             elif isinstance(val, (int, float, bool)):
                 update_ops = [dict(op="AddUpdateAttribute", attributeName=name, addUpdateAttribute=val)]
+            elif hasattr(val, "keys") and hasattr(val, "items"):  # TODO: better way to detect dict-like objects?
```

Comment:
I believe that `collections.Mapping` would probably work here (e.g. `isinstance(val, Mapping)`).  Dictionaries type as `collections.Mapping` and it seems to define both items and keys: https://docs.python.org/3.1/library/collections.html#collections.Mapping

Reply:
Edit: This is `collections.abc` now apparently: https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

Reply:
Good suggestion! I went with [typing.Mapping](https://docs.python.org/3/library/typing.html), which points to the collections definition.